tests/BulkTest.php

```diff
@@ -374,19 +374,22 @@ public function testRawDocumentDataRequest(): void
             new Document(null, '{"name":"The Human Torch"}'),
         ];
 
-        $bulk = new Bulk($client);
-        $bulk->addDocuments($documents);
-        $bulk->setIndex($index);
+        $bulk = (new Bulk($client))
+            ->setIndex($index)
+            ->addDocuments($documents)
+        ;
 
-        $expectedJson = '{"index":{}}
+        $expectedJson = <<<'JSON'
```

Comment:
What is the `JSON` used here for? Haven't seen this before.

Reply:
That's just HEREDOC syntax with `JSON` identifier ;)

Reply:
Ah, got it 👍